### ISSUE TITLE
Possible support for EmuFlight?

### ISSUE THREAD

**Issue by VitroidFPV** [bsondermann/BlackboxSticksExporter#5]
Hello, 
I recently switched from Betaflight to EmuFlight to see which might suit me more, and I noticed that the renders don't work properly. As in that, the throttle indicator doesn't appear at all in the renders. I tried it with a log that I know is from BF, and that worked properly.

**Comment by bsondermann** [comment]
Hi, can you send me an Emuflight log?

**Comment by VitroidFPV** [comment]
I don't know if the previous file reached you, Gmail said it was too large.
So just in case, I'll send a download link:
https://drive.google.com/file/d/1eVDJ-Sx57O-58025B9mzwF8nmYo-3IZK/view?usp=drivesdk



Dne so 2. 1. 2021 19:37 uživatel bsondermann <notifications@github.com>
napsal:

**Comment by bsondermann** [comment]
You need to make the file public to everyone or you need to give me access :).

**Comment by VitroidFPV** [comment]
Oh my God I'm so sorry 😅. Yea it should work now I think. I need to get the hang of all this Google stuff https://drive.google.com/file/d/1eVDJ-Sx57O-58025B9mzwF8nmYo-3IZK/view?usp=drivesdk

**Comment by bsondermann** [comment]
Ah nice, now it works :D. I will take a look asap.

**Comment by VitroidFPV** [comment]
Great, thanks!


Dne ne 3. 1. 2021 0:50 uživatel bsondermann <notifications@github.com>
napsal:

**Comment by bsondermann** [comment]
Hi, just released a new update, it should work now. Please give it a try :)

**Comment by VitroidFPV** [comment]
Yep! Works perfectly. Now if I just have some more suggestions, is it better to open another issue, or put it in this thread?

**Comment by bsondermann** [comment]
Just put it in this thread for now 👍

**Comment by VitroidFPV** [comment]
Okay then! I think the option to turn off certain elements would make for a cleaner result for some. Like the position values and the mode. As well as a way to maybe make the position "dot" smaller. With smaller movements, it is harder to notice them

**Comment by bsondermann** [comment]
Thanks for the feedback, I think I will add this in the future but it might take me 2 weeks because currently I am focusing in the 3D version of the exporter. You might want to give that a try ;) it looks awesome in my opinion!

**Comment by VitroidFPV** [comment]
It really does! In the future I can totally imagine it being used a ton. (might be a bit too far fetched, but imagine if you could choose different radios to virtualize)

**Comment by bsondermann** [comment]
I want to make an option where you can load custom textures on the 3D model but I first want to have more important stuff done :D
